core-feature-pack/galleon-common/src/main/resources/feature_groups/core-basic-profile.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<feature-group-spec name="basic-profile" xmlns="urn:jboss:galleon:feature-group:1.0">
+<feature-group-spec name="core-basic-profile" xmlns="urn:jboss:galleon:feature-group:1.0">
```

Comment:
the rule of thumb is that feature-groups which share the same name in WildFly (Full) and WildFly Core should be renamed to `core-xxx` to disambiguate. Is that correct?

Reply:
Yes. 

A simple way to think of it is all these shared galleon files end up being unzipped or copied into the same target/resources folder, so the same file in WF will overwrite the one from core. So to pick up the content from core either the core file needs a unique name, or all the content from the core file would have to be duplicated in the WF file.